.yamato/build_osx_x64arm64.yaml

```diff
@@ -0,0 +1,33 @@
+{% metadata_file .yamato/Globals.metafile %}
+---
+name: Generate OSX x64Arm64
+
+agent:
+  type: Unity::VM::osx
+  image: platform-foundation/mac-10.15-bokken:latest
```

Comment:
Do we want to avoid use of the `latest` tag? That has bitten us in other places where the image might change without our knowledge. Is there a `stable` tag we can use instead?

Reply:
fixed in 6c5334e27adadd68a93f76fb92338533c5beb6e6